state/backups/backups.go

```diff
@@ -138,11 +138,15 @@ func (b *backups) Create(meta *Metadata, paths *Paths, dbInfo *DBInfo) error {
 	if err != nil {
 		return errors.Annotate(err, "while listing files to back up")
 	}
+
+	backupDir, err:= "", nil
```

Comment:
TODO: Create a function that will return the actual `backupDir` using the method `BackupDir` form `./environs/config/config.go`.